mlton/ssa/ssa-tree.sig

```diff
@@ -209,6 +218,9 @@ signature SSA_TREE =
                      statements: Statement.t vector,
                      transfer: Transfer.t}
 
+	    val check: (int * int option) -> bool
+            val blockSize: int * int option -> (Exp.t -> int) * (Transfer.t -> int) -> t -> int * bool
+            val blocksSize: int * int option -> (Exp.t -> int) * (Transfer.t -> int) -> t vector -> int * bool
```

Comment:
It seems that clients never use the short-circuiting block size functions, so it should suffice to export:
```
  val size: t * {expSize: Exp.t -> int, transferSize: Transfer.t -> int} -> int
  val sizeV: t vector * {expSize: Exp.t -> int, transferSize: Transfer.t -> int} -> int
```
Because the specification will be viewed without the implementation, it is better to use descriptive record types.